addons/mail/static/src/models/activity_mark_done_popover_view/activity_mark_done_popover_view.js

```diff
@@ -37,10 +37,14 @@ registerModel({
         /**
          * Handles click on this "Done & Schedule Next" button.
          */
-        onClickDoneAndScheduleNext() {
-            this.activityViewOwner.activity.markAsDoneAndScheduleNext({
+        async onClickDoneAndScheduleNext() {
+            await this.activityViewOwner.activity.markAsDoneAndScheduleNext({
                 feedback: this.feedbackTextareaRef.el.value,
             });
+            if (!this.exists() || !this.component) {
```

Comment:
This condition (the `exists` part) will always fail, as the activity is deleted in `markAsDoneAndScheduleNext`, leading do the fix not being applied.

What actually needs to be done is to save the value before the method, probably `const { chatter } = this.activityViewOwner.activityBoxView`, and then `reloadParentView` that I explained before could be used.

Reply:
The one in `onClickDone` is probably broken too and should be adapted the same way.